### PR TITLE
Add detailed replication statistics

### PR DESCRIPTION
**Pull request by Sascha8a**

### CODE REVIEW

**Comment by pashagolub** [comment]
Hey @Sascha8a, should we close or update this PR?

Thanks in advance!

**Comment by Sascha8a** [comment]
I think it is still a valid dashboard and addition to the project. We should take another look at this.

File: pgwatch2/metrics/replication/10/metric_master.sql [hunk 1]
```diff
@@ -1,7 +1,13 @@
 SELECT
   (extract(epoch from now()) * 1e9)::int8 as epoch_ns,
   application_name as tag_application_name,
+  *,
```
Comment (pashagolub) [inline pgwatch2/metrics/replication/10/metric_master.sql:4]: Why do we need to add a star here? I suppose we should definitely list all needed columns instead

**Comment by pashagolub** [comment]
Outdated. Thanks for the efforts @Sascha8a. Feel free to reopen this PR in case of need.

**Pull request closed without merging**